core/Settings/Setting.php

```diff
@@ -147,8 +147,6 @@
 
     protected $key;
     protected $name;
-    protected $writableByCurrentUser = false;
```

Comment:
I refactored this Setting code as a `UserSetting` test failed. It was needed to change those 3 Settings classes to make sure a `UserSetting` does not create a DB connection when the constructor is called (as this a problem in Tracker mode).